File: C/1011.c
#include <stdio.h>
#include <math.h>
#define PI 3.14159

int main() {
    double n1;
    scanf("%lf", &n1);

    double resultado = (4.0/3.0) * PI * pow( n1, 3 );

    printf("VOLUME = %.3lf\n", resultado);

    return 0;
}

File: JavaScript/1011.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
let lines = input.split('\n');

let raio = parseFloat(lines.shift());

let volume = 4.0/3.0 * 3.14159 * Math.pow(raio, 3);

console.log(`VOLUME = ${volume.toFixed(3)}`);